test/common/numa_helpers.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// This file contains helpers for tests for UMF pool API
```

Comment:
I guess this could be updated (you can add an empty line between the license and the description)